components/Markets/ConditionalMarketCard.tsx

```diff
@@ -364,9 +364,11 @@ export function ConditionalMarketCard({ isPassMarket = false }: { isPassMarket?:
                 type="number"
                 value={amount || ''}
                 defaultValue={amount || ''}
+                rightSectionWidth={70}
                 rightSection={
                   <ActionIcon
-                    radius="md"
+                    w="80%"
+                    radius="lg"
```

Comment:
I'd more put radius="sm" so it fits container

Reply:
Will fix